docker-compose.yaml

```diff
@@ -62,6 +62,8 @@ services:
         - network.host=0.0.0.0
         - bootstrap.memory_lock=true
         - "ES_JAVA_OPTS=-Xms1536m -Xmx1536m"  # Ensure Docker is allocated plenty of memory, otherwise this will fail
+      # Inject plugin install, then resume with orignial entrypoint command
+      command: /bin/sh -c "./bin/elasticsearch-plugin install mapper-murmur3 && /usr/local/bin/docker-entrypoint.sh"
```

Comment:
Most devs aren't on docker compose. Does this change imply a change in the non compose testing protocol that needs to be documented?

Reply:
Well, yes, but -- this _**is**_ the currently documented protocol. i.e., to run tests, its implied you're using the "Elasticsearch Setup" steps here: https://github.com/fedspendingtransparency/usaspending-api#elasticsearch-setup

(Is there another protocol documented somewhere else??)

I'll reply back with a the commit to add a line in the ["Test Setup"](https://github.com/fedspendingtransparency/usaspending-api#test-setup) section to make this explicit rather than implied.

And we version control this configuration for spinning up a local ES cluster to aid with the running of tests locally.

Also, fwiw, this was not germane to this PR, but really is something that should have been added to @sethstoudenmier PR: https://github.com/fedspendingtransparency/usaspending-api/pull/2349

Reply:
Looks like our ES documentation has long moved to the assumption that everyone is using docker-compose for everything. In my experience of the team that is fantasy, but that's not a concern for this PR.

Reply:
Added: https://github.com/fedspendingtransparency/usaspending-api/pull/2357/commits/cc1fa3f713c36fd1d83bb3c64ffcf2be5df769ff

Reply:
>  In my experience of the team that is fantasy, but that's not a concern for this PR.

@BrianZito After the continued improvements of the setup steps, I am unaware of what is blocking people from using Docker/docker-compose for testing. Docker compose is necessary to run the entire integration test suite since it relies on spinning up a local Broker instance.

Reply:
Might be nothing but social inertia. I'll go ahead and put a topic on the backlog for dev meetings.